tensorflow_datasets/image/caltech_birds.py

```diff
@@ -57,9 +57,9 @@ class CaltechBirds2010(tfds.core.GeneratorBasedBuilder):
   def _caltech_birds_info(self):
     return CaltechBirdsInfo(
         name=self.name,
-        images_url="http://www.vision.caltech.edu/visipedia-data/CUB-200/images.tgz",
-        split_url="http://www.vision.caltech.edu/visipedia-data/CUB-200/lists.tgz",
-        annotations_url="http://www.vision.caltech.edu/visipedia-data/CUB-200/annotations.tgz",
+        images_url="https://www.vision.caltech.edu/visipedia-data/CUB-200/images.tgz",
```

Comment:
Are you sure you tested this correctly ? I'm getting "website can't be accessed" with this new URL. I haven't checked the other ones.

Reply:
This is weird :smile: perhaps it was a mistake.. It's not working now, I'll revert this line